js/tests/unit/modal.js

```diff
@@ -29,6 +29,8 @@ $(function () {
       $.fn.bootstrapModal = $.fn.modal.noConflict()
     },
     afterEach: function () {
+      $('.modal-backdrop, #modal-test').remove()
+      $(document.body).removeClass('modal-open')
```

Comment:
I added that because in our `index.html` which run our unit test we did the same see : https://github.com/twbs/bootstrap/blob/v4-dev/js/tests/index.html#L82

Reply:
But do we need to do it in modal.js if we do it in index.html?

Reply:
Karma do not use our index.html it generate a new index.html